snappy_pipeline/workflows/somatic_msi_calling/__init__.py

```diff
@@ -60,30 +61,48 @@
 
 __author__ = "Clemens Messerschmidt <clemens.messerschmidt@bih-charite.de>"
 
+#: Extensions of files to create as main payload
+EXT_VALUES = (".results.txt", ".results.txt.status", ".results.txt.md5", ".results.txt.status.md5")
+
+#: Names of the files to create for the extension
+EXT_NAMES = ("result", "status", "result_md5", "status_md5")
+
+EXT_MATCHED = {
+    "mantis_msi2": {
```

Comment:
What is the format of the output? What is `result`, and what is `status`? Ideally, you want a 3rd output containing the minimal MSI/not MSI status in a format independent of the caller. That way, downstream steps can collect the main information without having to rely on the tool's specifics.